tools/schemacode/schemacode/render.py

```diff
@@ -527,7 +542,7 @@ def make_metadata_table(schema, field_info, tablefmt="github"):
     return table_str
 
 
-def make_columns_table(schema, column_info, tablefmt="github"):
+def make_columns_table(schema, column_info, tablefmt="github", relpath=None):
```

Comment:
Inconsistent ordering.
```suggestion
def make_columns_table(schema, column_info, relpath=None, tablefmt="github"):
```

Reply:
thanks, fixed in https://github.com/bids-standard/bids-specification/pull/1056/commits/3b3547bfd2f9d5cfb052a4724272240eb3d43af7